sdk/resourcemanager/Proto.Client/src/Scenarios/CreateVMSS.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using Azure.Identity;
+using Azure.ResourceManager.Core;
+using Proto.Compute;
+using Proto.Network;
+
+namespace Proto.Client
+{
+    class CreateVMSS : Scenario
```

Comment:
is this scenario added to the enum?

Reply:
this is not fully working. Skipping for now.